src/sentry/api/endpoints/organization_onboarding_tasks.py

```diff
@@ -8,14 +8,15 @@
 )
 
 from sentry.models import OnboardingTaskStatus, OrganizationOnboardingTask
+from sentry.receivers import check_for_onboarding_complete
 
 
 class OrganizationOnboardingTaskEndpoint(OrganizationEndpoint):
     permission_classes = [OrganizationPermission, ]
 
     def post(self, request, organization):
-        if request.DATA['status'] == 'skipped':
-            OrganizationOnboardingTask.objects.create_or_update(
+        if request.DATA['status'] == 'skipped' and int(request.DATA['task']) in [9, 10]:
```

Comment:
could 9,10 be swapped out w/ constants from the model so its more clear what they mean